doc/en/api/1.0.0/importer.yaml

```diff
@@ -99,14 +127,585 @@ paths:
+      bbox:
+        $ref: "#/definitions/Bbox"
+      attributes:
+        $ref: "#/definitions/FeatureType"
+      style:
+        $ref: "#/definitions/Style"
+  FeatureType:
+    title: featureType
+    type: array
+    description: Feature type attributes
+    items:
+      type: object
+      description: A feature type attribute.
+      properties:
+        name:
+          type: string
+          description: The name of the attribute
+        binding:
+          type: string
+          description: The java class representing the type of the attribute
+  Style:
+    title: style
+    type: object
+    description: A layer style
+    properties:
+      name:
+        type: string
+        description: Name of the style
+      href:
+        type: string
+        description: "URL to the importer layer style endpoint"
+      format:
+        $ref: "styles.yaml/definitions/Style/properties/format"
+      languageVersion:
+        $ref: "styles.yaml/definitions/Style/properties/languageVersion"
+      filename:
+        $ref: "styles.yaml/definitions/Style/properties/filename"
+  TransformChain:
+    title: transformChain
+    type: object
+    description: A chain of tranform operations to apply during the import process
+    properties:
+      type:
+        type: string
+        description: 'The type of transforms in the chain. One of "vector" or "raster"'
+      transforms:
+        $ref: "#/definitions/Transforms"
+  Transforms:
+    title: transforms
+    type: array
+    description: A list of transforms
+    items:
+      $ref: "#/definitions/Transform"
+  Transform:
+    title: transform
+    type: object
+    description: A data transform applied to the import data.
+    properties:
+      type:
+        type: string
+        description: The name of the transform
+      href: 
+        type: string
+        description: URL to the transform endpoint
+#  Transform:
+#    oneOf:
+#      - $ref: "#/definitions/DataFormatTransform"
+#      - $ref: "#/definitions/IntegerFieldToDateTransform"
+#      - $ref: "#/definitions/CreateIndexTransform"
+#      - $ref: "#/definitions/AttributeRemapTransform"
+#      - $ref: "#/definitions/AttributesToPointGeometryTransform"
+#.     - $ref: "#/definitions/ReprojectTransform"
+#      - $ref: "#/definitions/GdalTranslateTransform"
+#      - $ref: "#/definitions/GdalWarpTransform"
+#      - $ref: "#/definitions/GdalAddoTransform"
+  DataFormatTransform:
+    title: DataFormatTransform
+    type: object
+    description: A data transform that converts a non date attribute in a date attribute.
+    properties:
+      type:
+        type: string
+        description: '"DataFormatTransform"'
+      href: 
+        type: string
+        description: URL to the transform endpoint
+      field:
+        type: string
+        description: The field to apply the transform to
+      format: 
+        type: string
+        description: A format string used to construct the transformed date
```

Comment:
Doesn't look like it. The pattern is some custom regex-based pattern written by justin, and seems to be largely undocumented.

I found this in the old importer rest docs, which I will add, but still no good examples:
> A date parsing pattern, setup using the Java SimpleDateFormat syntax. In case it’s missing, a number of built-in formats will be tried instead (short and full ISO date formats, dates without any separators).